gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStoragePathTest.java

```diff
@@ -46,75 +46,74 @@
 
   @Rule public final ExpectedException thrown = ExpectedException.none();
 
-  @Before
-  public void before() {
-    CloudStorageFileSystemProvider.setGCloudOptions(LocalGcsHelper.options());
-  }
+  private final FakeStorageRpc storage = new FakeStorageRpc(true);
+  private final NioTestHelper helper = new NioTestHelper(storage);
 
   @Test
-  public void testCreate_neverRemoveExtraSlashes() {
-    try (CloudStorageFileSystem fs = forBucket("doodle")) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/mziccard/gcloud-java/file/2334092444/issues/source?bid=3123107&fileBranchId=3190440#l53)